src/machine/machine_atsamd21g18.go

```diff
@@ -294,3 +295,115 @@ func (p GPIO) setPinCfg(val sam.RegValue8) {
 		sam.PORT.PINCFG0_31 = val
 	}
 }
+
+// UART
+var (
+	// The first hardware serial port on the SAMD21. Uses the SERCOM0 interface.
+	UART0 = &UART{}
+)
+
+const (
+	SAMPLE_RATE_x16           = 16
+	LSB_FIRST                 = 1
+	SERCOM_RX_PAD_0           = 0
+	SERCOM_RX_PAD_1           = 1
+	SERCOM_RX_PAD_2           = 2
+	SERCOM_RX_PAD_3           = 3
+	UART_TX_PAD_0             = 0 // Only for UART
+	UART_TX_PAD_2             = 1 // Only for UART
+	UART_TX_RTS_CTS_PAD_0_2_3 = 2 // Only for UART with TX on PAD0, RTS on PAD2 and CTS on PAD3
```

Comment:
What are these constants? Are they meant to be exported?

Reply:
These are some bitfields that are not generated due to not being in the SVD file. They are not really meant to be exported, but they are needed or will be needed for other SERCOM uses such as the I2C and SPI interfaces.

Perhaps they need to be identified as such? They are the same values for all members in the SAMD21 family of devices. Maybe we need a file `samd21-bitfields.go` to be force committed to `devices/sam` in similar fashion to how we handled some of the stm32 bitfields.

Reply:
> Maybe we need a file `samd21-bitfields.go` to be force committed to `devices/sam` in similar fashion to how we handled some of the stm32 bitfields.

￼What you prefer. The important thing for me is that these fields are not exported, because they are not part of the public API of the machine module.

Reply:
That makes sense. Change has been pushed.